doc/source/contribute.rst

```diff
@@ -68,13 +68,24 @@ At this point, wait for reviewer approval and conclude any discussion that arise
 Release checklist
 ~~~~~~~~~~~~~~~~~
 
-- [ ] Check that the [build is passing](https://travis-ci.org/kivy/python-for-android)
+- [ ] Check that the builds are passing
+
+  - [ ] `Travis build <https://travis-ci.org/kivy/python-for-android>`_
+
+  - [ ] `GitHub Action <https://github.com/kivy/python-for-android/actions>`_
+
 - [ ] Run the tests locally via `tox`: this performs some long-running tests that are skipped on Travis.
-- [ ] Build and run the [on_device_unit_tests](https://github.com/kivy/python-for-android/tree/master/testapps/on_device_unit_tests) app using buildozer. Check that they all pass.
-- [ ] Build and run the following [testapps](https://github.com/kivy/python-for-android/tree/master/testapps) for arch `armeabi-v7a` and `arm64-v8a`:
-  - `python3 setup_testapp_python3_sqlite_openssl.py apk`
-    - [ ] `armeabi-v7a`
-    - [ ] `arm64-v8a`
+
+- [ ] Build and run the `on_device_unit_tests <https://github.com/kivy/python-for-android/tree/master/testapps/on_device_unit_tests>`_ app using buildozer. Check that they all pass.
+
+- [ ] Build (or download from github actions) and run the following `testapps <https://github.com/kivy/python-for-android/tree/master/testapps/on_device_unit_tests>`_ for arch `armeabi-v7a` and `arm64-v8a`:
+
+  - [ ] on_device_unit_tests
+
+    - [ ] `armeabi-v7a` (cmd: `python3 testapps/on_device_unit_tests/setup.py  --arch=armeabi-v7a --debug`)
+
+    - [ ] `arm64-v8a` (cmd: `python3 testapps/on_device_unit_tests/setup.py  --arch=arm64-v8a --debug`)
```

Comment:
Mmmm..my mistake in here, I realized that I gived to you a wrong command, **sorry** :grimacing:, this is more appropiated:


```suggestion
    - [ ] `armeabi-v7a`::

          cd testapps/on_device_unit_tests && PYTHONPATH=. python3 setup.py apk  --ndk-dir=<your-ndk-dir> --sdk-dir=<your-sdk-dir> --arch=armeabi-v7a --debug

    - [ ] `arm64-v8a`::

          cd testapps/on_device_unit_tests && PYTHONPATH=. python3 setup.py apk  --ndk-dir=<your-ndk-dir> --sdk-dir=<your-sdk-dir> --arch=arm64-v8a --debug
```

Reply:
I actually had some doubt as I thought I tried in the past with `cd`-ing and we couldn't.
Anyway, now I'm think we should maybe keep this simple and not give the command example at all

Reply:
I like including the command example because the intention of this doc is to make preparing a release as absolutely thoughtless as possible. Perhaps it would be nice to move it to a makefile command that builds the release targets and prints some clear information about where they were output (but of course don't feel the need to do that here).

Reply:
Ei guys I fixed [the command](https://github.com/kivy/python-for-android/pull/2177#discussion_r422480998): PYTHONPATH should include p4a sources and testapp sources:

```PYTHONPATH=.:../../```

That is why didn't work for you @AndreMiras